torchvision/ops/_register_onnx_ops.py

```diff
@@ -20,8 +20,8 @@ def symbolic_multi_label_nms(g, boxes, scores, iou_threshold):
 
     @parse_args('v', 'v', 'f', 'i', 'i', 'i', 'i')
     def roi_align(g, input, rois, spatial_scale, pooled_height, pooled_width, sampling_ratio, aligned):
-        if(aligned):
-            raise RuntimeError('Unsupported: ONNX export of roi_align with aligned')
+        if aligned:
+            rois = g.op("Sub", rois, torch.tensor(0.5))
```

Comment:
This is unfortunately not all that needs to be changed so that the implementation supports the `aligned` flag.

Indeed, this only works if the spatial_scale is 1, and if the boxes are not malformed.

See https://github.com/pytorch/vision/blob/3e0f5a6ff24c4be25cdaa6695bad421c99e257b9/torchvision/csrc/cpu/ROIAlign_cpu.cpp#L138-L150 for the locations where the `aligned` flag plays a role. I believe we would need at least to  do something like
```python
if aligned:
    rois = (rois * spatial_scale - 0.5) / spatial_scale
    # or equivalently
    rois = rois - 0.5 / spatial_scale
```
but the difference wrt the clamping in https://github.com/pytorch/vision/blob/3e0f5a6ff24c4be25cdaa6695bad421c99e257b9/torchvision/csrc/cpu/ROIAlign_cpu.cpp#L146-L150 persists

Reply:
Thanks a lot for reviewing this. It was very helpful.
Since ONNX forces all ROIs to be 1x1 or larger, I think fixing the export for malformed ROIs needs a change in spec.